### PR TITLE
Replace suite#show preview image on doc site

### PR DESCRIPTION
**Pull request by meghan-kradolfer**
The TA docs are pretty outdated, this has started to cause some issues recently with customers wondering why their product doesn't look like the docs. https://3.basecamp.com/3453178/buckets/20365997/card_tables/cards/6803183541

I did a quick scan of bits that referenced features that have been completely removed and updated them. However I do think this needs an full check/rewrite at some point

### CODE REVIEW

File: pages/test_analytics/test_suites.md
Comment (meghan-kradolfer) [inline pages/test_analytics/test_suites.md:75]: I feel like this also needs to be updated, we no longer show "failed test execution count" and I don't believe we have "detailed information about span duration" on the test page?

Comment (preflightcookie) [reply pages/test_analytics/test_suites.md:75]: I agree. There is info on failures, you can toggle the tab between flakies and fails, but not a count. No span data on this page.

File: pages/test_analytics/test_suites.md
Comment (preflightcookie) [inline pages/test_analytics/test_suites.md:77]: Doesn't include failed execution count

```suggestion
<%= image "test-stats.png", width: 1166, height: 327, alt: "Screenshot of test trend page showing test trend information over the last 28 days, including reliability and test execution durations" %>
```

Comment (meghan-kradolfer) [reply pages/test_analytics/test_suites.md]: Done!

**Comment by KatieWright26** [comment]
Are you able to merge this now @meghan-kradolfer ?

**Comment by meghan-kradolfer** [comment]
> Are you able to merge this now @meghan-kradolfer ?

Just waiting for a review from @mbelton-buildkite

File: images/docs/test_analytics/test_suites/test-stats.png
Comment (mbelton-buildkite) [inline images/docs/test_analytics/test_suites/test-stats.png:1]: Should the sentence above the stats make sense? I can't parse it 😅

**Review by mbelton-buildkite — APPROVED** [review]
One comment about the screenshot, but looks good otherwise 😊

File: images/docs/test_analytics/test_suites/test-stats.png
Comment (meghan-kradolfer) [reply images/docs/test_analytics/test_suites/test-stats.png:1]: That is just the test 'name' which we generate using the test scope and test name. I might replace with one that actually reads well!

Comment (mbelton-buildkite) [reply images/docs/test_analytics/test_suites/test-stats.png:1]: 👍🏻 Thanks!

**Pull request merged**
